README.md

```diff
@@ -0,0 +1,3 @@
+## Interview Question Solutions

```

Comment:
We have to change the title to something else

Reply:
Why? Isn't this what this repo is about? If not what's another name that you want to change it to and its rationale?

Reply:
Nee publicly ente kaiyil ninnu adi vangikkaruthu. Change this to some java examples or something!